<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>countdown_timer::TimerSettingsWidget</class>
 <widget class="QWidget" name="countdown_timer::TimerSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="6" column="0" colspan="2">
    <widget class="QTextEdit" name="msg_text_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="reverse_counting">
     <property name="text">
      <string>reverse counting</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="hide_if_inactive">
     <property name="text">
      <string>hide timer when it is not active</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="show_msg">
     <property name="text">
      <string>show custom message on timeout</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="restart_on_timeout">
     <property name="text">
      <string>auto-restart countdown</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="restart_on_dblclik">
     <property name="text">
      <string>restart on double click</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="timeout_frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
      <item row="0" column="0">
       <widget class="QRadioButton" name="target_time_rbtn">
        <property name="text">
         <string>target time</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QDateTimeEdit" name="target_time_edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="interval_rbtn">
        <property name="text">
         <string>interval</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="h_edit">
        <property name="toolTip">
         <string>hours</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="m_edit">
        <property name="toolTip">
         <string>minutes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="s_edit">
        <property name="toolTip">
         <string>seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="show_leading_zero">
       <property name="text">
        <string>leading zero</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="min_days_label">
       <property name="text">
        <string>minimum days count to show</string>
       </property>
       <property name="buddy">
        <cstring>min_days_edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="min_days_edit">
       <property name="toolTip">
        <string>If remaining days count less than this value, days will not be shown.
Set this to -1 to never show days count, or to 0 to always show.</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="also_hide_hours">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>also hide hours</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chime_on_timeout">
       <property name="text">
        <string>chime when countdown finishes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="browse_sound_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>target_time_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>target_time_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interval_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>h_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interval_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interval_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>s_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interval_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>restart_on_dblclik</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interval_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>restart_on_timeout</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chime_on_timeout</sender>
   <signal>toggled(bool)</signal>
   <receiver>browse_sound_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_msg</sender>
   <signal>toggled(bool)</signal>
   <receiver>msg_text_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
